package/MDAnalysis/coordinates/memory.py

```diff
@@ -502,8 +502,10 @@ def timeseries(self, asel=None, start=0, stop=-1, step=1, order='afc'):
             data is returned whenever `asel` is different from ``None``.
         start : int (optional)
         stop : int (optional)
+            .. deprecated:: 3.0
+                Note that `stop` is currently *inclusive* but will be
+                deprecated in favour of being *exclusive* in 3.0.  
```

Comment:
```suggestion
            .. deprecated:: 2.4.0
                Note that `stop` is currently *inclusive* but will be
                changed in favour of being *exclusive* in version 3.0.  
```
essentially deprecation happens now, change happens in the future